src/amp-shadow.js

```diff
@@ -64,7 +65,7 @@ if (isExperimentOn(self, 'ampdoc-shell')) {
   const ampdocShell = ampdocService.installShellShadowDoc();
   installStylesForDoc(
     ampdocShell,
-    cssText,
+    ampDocCss + ampElementCss,
```

Comment:
Just FYI, for now I prefer this way. But if we hit some ordering complications, I believe there's also an option to do `@import` in CSS.

Reply:
this way allows us to do easy client side experiment

Reply:
Agree. My hope this would be fine at this stage.